CONTRIBUTING.md

```diff
@@ -64,14 +64,14 @@ Right now we're working on a list of things we want help with and easy-to-fix bu
 
 0. Associate your local git config with your github account. If this is your first time using git you can follow [the steps](#associate-with-github-account).
```

Comment:
```suggestion
1. Associate your local git config with your github account. If this is your first time using git you can follow [the steps](#associate-with-github-account).
```

Reply:
It was originally kept 0 and so I let it be to give special importance. The rest of the numbering is working fine.